README.md

```diff
@@ -1 +1,6 @@
-# nmpolicy
\ No newline at end of file
+# nmpolicy
+
+## Release
+
+To do a new release push a semVer tag to the repo, the github action will run 
+gorelease to collect changelogs and generate artifacts.
```

Comment:
Usually the README file is for consumers of the project and not for developers or contributors.
Can you place this under a different file like some of the other projects do?

It is also preferred to add the exact procedure that one needs to do in order to release. See example [here](https://github.com/networkplumbing/go-nft/blob/main/README.release.md).